src/components/ha-drawer.ts

```diff
@@ -9,6 +14,11 @@ const blockingElements = (document as any).$blockingElements;
 
 @customElement("ha-drawer")
 export class HaDrawer extends DrawerBase {
+  firstUpdated() {
+    super.firstUpdated();
+    this.mdcRoot.dir = document.dir;
+  }
```

Comment:
What if the user change the language ? A page refresh is required ?

Reply:
> What if the user changes the language ? A page refresh is required ?

Yeah, it is for now... I thought it was better than nothing 🙃

Reply:
Fixed